californium-core/src/main/java/org/eclipse/californium/core/network/Exchange.java

```diff
@@ -203,6 +203,10 @@ public Origin getOrigin() {
 		return origin;
 	}
 
+    public boolean isOfLocalOrigin() {
+        return origin == Origin.LOCAL;
+    }
+
```

Comment:
Indentation?
You can use the eclipse formatter rules contained in the parent project if you are using eclipse ...